releases/1.0.0.md

```diff
@@ -0,0 +1,73 @@
+## Arches Modular Reports 1.0.0 beta Release Notes
+
+## How to setup Arches Modular Reports for your project
```

Comment:
Do we want to maintain this in two places?

Reply:
I was really just cribbing from AfS on their initial release.  It is redundant.